broker-core/src/main/java/org/eclipse/kapua/broker/core/plugin/authentication/AuthenticationLogic.java

```diff
@@ -165,12 +166,12 @@ protected void enforceDeviceConnectionUserBound(Map<String, Object> options, Dev
         if (deviceConnection != null) {
             ConnectionUserCouplingMode connectionUserCouplingMode = deviceConnection.getUserCouplingMode();
             if (ConnectionUserCouplingMode.INHERITED.equals(deviceConnection.getUserCouplingMode())) {
-                connectionUserCouplingMode = loadConnectionUserCouplingModeFromConfig(scopeId, options);
+                connectionUserCouplingMode = loadConnectionUserCouplingModeFromConfig(options);
```

Comment:
did you change the loadConnectionUserCouplingModeFromConfig?
isn't it no more bound to a scope?

Reply:
`scopeId` is not used in `loadConnectionUserCouplingModeFromConfig()`, so it has been removed from the method signature.

See AuthenticationLogic.java:246